app/routers/v1/OrganizationRouter.js

```diff
@@ -0,0 +1,11 @@
+// OrganizationRouter
+module.exports = exports = {
+  '/organizations' : {
```

Comment:
I have just remembered that you can instead of specifying the individual routes, you can say, use this resource controller and then using the 'allow' or 'deny' key, prevent certain routes.  Set up the route as a resource using the OrganizationController, but add a 'deny: ['create', 'delete'] field under the controller field.  Please see documentation for example.  As a note, I will be making an issue to change the UserRouters to do this as well.